osu.Game/Localisation/SkinEditorStrings.cs

```diff
@@ -42,7 +42,17 @@ public static class SkinEditorStrings
         /// <summary>
         /// "Currently editing"
         /// </summary>
-        public static LocalisableString CurrentlyEditing => new TranslatableString(getKey(@"currently_editing"), "Currently editing");
+        public static LocalisableString CurrentlyEditing => new TranslatableString(getKey(@"currently_editing"), @"Currently editing");
+
+        /// <summary>
+        /// "Revert to default?"
+        /// </summary>
+        public static LocalisableString RevertToDefault => new TranslatableString(getKey(@"revert_to_default"), @"Revert to default?");
+
+        /// <summary>
+        /// "The skin will return to the state it was in upon import"
+        /// </summary>
+        public static LocalisableString ResetDialogue => new TranslatableString(getKey(@"the_skin_will_return_to"), @"The skin will return to the state it was in upon import");
```

Comment:
The bigger issue here is that this text doesn't even make sense in all usages. Let's say you take the default skin, start messing with it, but then decide to reset to default. What does "the state it was in upon import" even mean in that case? The skin was never imported.

Reply:
any suggestions? "return the skin to its default state" perhaps?, or some sort of detection for triangles and argon to  change the message for them?

Reply:
Default state is fine.